src/lib/transaction/Transaction.svelte

```diff
@@ -34,6 +35,28 @@
             let operationComponent = dynamicOperationComponentFactory.create(tx, tx.operations[i]!);
             operationComponents.push(operationComponent);
         }
+
+        if (
+            txParams.operationsGroups.length > 0 &&
+            operationComponents[txParams.operationsGroups[txParams.operationsGroups.length - 1]!.to]
+        ) {
+            for (let i = 0; i < txParams.operationsGroups.length; i++) {
+                operationsGroups.push({
+                    description: txParams.operationsGroups[i]!.description,
+                    operationsComponents: operationComponents.slice(
+                        txParams.operationsGroups[i]!.from,
+                        txParams.operationsGroups[i]!.to + 1,
```

Comment:
this should not be repeated multiple times `txParams.operationsGroups[i]`

Reply:
Why do you need the `!` in the operationGroups element? I don't see it defined as nullable